roles/deploy/tasks/build.yml

```diff
@@ -20,5 +21,6 @@
   with_items: "{{ project_folder_paths.results }}"
   when: item.stat.exists
 
-- include: "{{ deploy_build_after | default('../hooks/example.yml') }}"
-  tags: deploy-build-after
+- include: "{{ item }}"
+  with_items: "{{ deploy_build_after | default([]) }}"
+  tags: deploy-build-after
```

Comment:
Could you add back the ending [newline](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) for these various files?